Description: Move the demos from /usr/share to /usr/lib
Author: Laurent Bigonville <bigon@debian.org>

--- a/demo/geoclue-where-am-i.desktop.in.in
+++ b/demo/geoclue-where-am-i.desktop.in.in
@@ -3,7 +3,7 @@ _Name=Where am I?
 _GenericName=Demo geolocation application
 _Comment=Find your current location
 _Keywords=geolocation;
-Exec=@libexecdir@/geoclue-2.0/demos/where-am-i
+Exec=/usr/lib/geoclue-2.0/demos/where-am-i
 Icon=mark-location-symbolic
 NoDisplay=true
 Terminal=true
